pkg/logs/input/docker/parser.go

```diff
@@ -139,3 +143,18 @@ func min(a, b int) int {
 	}
 	return b
 }
+
+func isEmptyMessage(content []byte) bool {
+	contentLen := len(content)
+
+	if contentLen == 2 && content[0] == '\\' {
+		switch content[1] {
+		case 'n', 'r':
+			return true
+		}
+	} else if bytes.Equal(content, escapedCRLF) {
```

Comment:
you could remove this else if and return directly `bytes.Equal(content, escapedCRLF)`